ckan/controllers/group.py

```diff
@@ -314,8 +314,14 @@ def pager_url(q=None, page=None):
 
             facets = OrderedDict()
 
-            default_facet_titles = {'organization': _('Organizations'),
-                                    'groups': _('Groups'),
+            org_type = h.default_group_type(u'organization') + u's'
+            org_label = h.humanize_entity_type(org_type).capitalize()
+
+            group_type = h.default_group_type(u'group') + u's'
```

Comment:
Let's leave pluralization in helpers that can be overridden by sites (like humanize_entity_type) because it's not the same for all words/languages